projects/plugins/jetpack/_inc/client/recommendations/sidebar/mobile-app/index.tsx

```diff
@@ -0,0 +1,195 @@
+				src={ imagePath + `/recommendations/mobile-app-qr-code-${ slug }.png` }
+				width={ 80 }
+				height={ 80 }
+				alt="qr code to jetpack.com/mobile/ web page"
+			/>
+			<p className="jp-recommendations-sidebar-mobile__qr-code-body">
+				{ createInterpolateElement(
+					__(
+						'Visit <ExternalLink>jetpack.com/mobile</ExternalLink> or scan this code to download the Jetpack mobile app.',
+						'jetpack'
+					),
+					{
+						ExternalLink: (
+							<ExternalLink
+								href={ getRedirectUrl( 'jetpack-plugin-recommendations-mobile-app-component' ) }
+								rel="noopener noreferrer"
+								target="_blank"
+								onClick={ onJpcomAppClick }
+							></ExternalLink>
+						),
+					}
+				) }
+			</p>
+		</div>
+	);
+
+	const getHeading = () => {
+		switch ( slug ) {
+			case 'download-app':
+				return __( 'Check your site activity anywhere, any time.', 'jetpack' );
+			case 'one-click-restores':
+				return __( 'Restore a backup anytime with the Jetpack mobile app.', 'jetpack' );
+			case 'manage-security':
+				return __( 'Realtime security notifications anywhere, any time.', 'jetpack' );
+			case 'upsell':
+				return __( 'Jetpack puts WordPress in your pocket.', 'jetpack' );
+			default:
+				return __( 'Jetpack puts WordPress in your pocket.', 'jetpack' );
+		}
+	};
+
+	const getBody = () => {
+		switch ( slug ) {
+			case 'download-app':
+				return __(
+					'Never miss an important event with realtime notifications and your activity log just a tap away.',
+					'jetpack'
+				);
+			case 'one-click-restores':
+				return __(
+					'Keep track of your site’s activity and restore your site anywhere, any time.',
+					'jetpack'
+				);
+			case 'manage-security':
+				return __(
+					'Get instant alerts for serious issues and fix them with a simple tap.',
+					'jetpack'
+				);
+			case 'upsell':
+				return __(
+					'The Jetpack mobile app gives you everything you need to grow and manage your WordPress site anywhere, anytime.',
+					'jetpack'
+				);
+			default:
+				return __(
+					'The Jetpack mobile app gives you everything you need to grow and manage your WordPress site anywhere, anytime.',
+					'jetpack'
+				);
+		}
+	};
+
+	// Get device type and display relevant link
+	// Google play store link for android, App Store link for ios, and QR code otherwise
+	const getAppLinkSection = () => {
+		const device = detectMobileDevice();
+
+		switch ( device ) {
+			case 'ios':
+				return <AppStoreBadge />;
+			case 'android':
+				return <GooglePlayBadge />;
+			case 'windows':
+			case 'unknown':
+				return <QRCode />;
+			default:
+				return <QRCode />;
+		}
+	};
+
+	const headerImgSize = underside ? 35 : 25;
+	const header = (
+		<div className="jp-recommendations-sidebar-mobile__header-container">
+			<img
+				src={ imagePath + '/recommendations/wordpress-icon.svg' }
+				width={ headerImgSize }
+				height={ headerImgSize }
+				alt="wordpress icon"
```

Comment:
Since icons are decorative here, I think it'd be a better experience for screen reader users to have an empty `alt` attribute.

Reply:
Agreed 😄